dangerfile.ts

```diff
@@ -18,26 +18,26 @@ import { distanceInWords } from "date-fns"
 
 // For some reason we're getting type errors on this includes module?
 // Wonder if we could move to the includes function in ES2015?
-import * as includesOriginal from "lodash.includes"
-const includes = includesOriginal as Function
-
+import includes from "lodash.includes"
 const sentence = danger.utils.sentence
```

Comment:
The above was a hack because the inferred type of `lodash.includes` was `object: collection, target: any, fromIndex?:any, somethingElse:any` having a extra mandatory 4th param. By adding `@types/lodash.include` it went to ``object: collection, target: any, fromIndex?:any`.